tutorial/numpy-functions.md

```diff
@@ -329,14 +329,16 @@ In Pyccel we try to support the Numpy functions which developers use the most..
 
 -   Supported types : bool, int, int8, int16, int32, int64, float, float32, float64, complex64 and complex128. they can be used as cast functions too.
```

Comment:
Small corrections:
- There should be no empty space between `types` and `:`
- `they` is at the beginning of the sentence and hence it should be capitalized: `They`